protocols/example/valueentity/shoppingcart/shoppingcart.proto

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 Lightbend Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This is the public API offered by the shopping cart entity.
+syntax = "proto3";
```

Comment:
@ralphlaude Do we need this file? Would it not be possible to use the pre-existing version of the shoppingcart.proto and implement it using ValueEntity (or EventSourcedEntity for our other demo/example).

Reply:
@viktorklang thanks for the hint and i will try to use the pre-exsiting shoppingcart.proto for the ValueEntity demo/example. the